### ISSUE TITLE
JWT Signature verification failed

### ISSUE THREAD

**Issue by pmatos** [buildbot/buildbot#3417]
Hello,

In the midst of the configuration of a new buildbot `0.9.6` with `python3`, I get the following error. Have absolutely no idea what this means. I understand the traceback might not be that useful but if you could, can you please let me know what else I can provide you to help track down the problem?

```
2017-07-06 14:31:33+0200 [_GenericHTTPChannelProtocol,0,127.0.0.1] while decoding JWT session
        Traceback (most recent call last):
          File "/home/pmatos/.pyenv/versions/3.6.0/envs/buildbot3-venv/lib/python3.6/site-packages/twisted/web/server.py", line 448, in getSession
            session = self.site.getSession(sessionCookie)
          File "/home/pmatos/.pyenv/versions/3.6.0/envs/buildbot3-venv/lib/python3.6/site-packages/buildbot/www/service.py", line 172, in getSession
            return BuildbotSession(self, uid)
          File "/home/pmatos/.pyenv/versions/3.6.0/envs/buildbot3-venv/lib/python3.6/site-packages/buildbot/www/service.py", line 74, in __init__
            self._fromToken(token)
          File "/home/pmatos/.pyenv/versions/3.6.0/envs/buildbot3-venv/lib/python3.6/site-packages/buildbot/www/service.py", line 87, in _fromToken
            log.err(e, "while decoding JWT session")
        --- <exception caught here> ---
          File "/home/pmatos/.pyenv/versions/3.6.0/envs/buildbot3-venv/lib/python3.6/site-packages/buildbot/www/service.py", line 83, in _fromToken
            decoded = jwt.decode(token, self.site.session_secret, algorithms=[SESSION_SECRET_ALGORITHM])
          File "/home/pmatos/.pyenv/versions/3.6.0/envs/buildbot3-venv/lib/python3.6/site-packages/jwt/api_jwt.py", line 64, in decode
            options, **kwargs)
          File "/home/pmatos/.pyenv/versions/3.6.0/envs/buildbot3-venv/lib/python3.6/site-packages/jwt/api_jws.py", line 126, in decode
            key, algorithms)
          File "/home/pmatos/.pyenv/versions/3.6.0/envs/buildbot3-venv/lib/python3.6/site-packages/jwt/api_jws.py", line 196, in _verify_signature
            raise DecodeError('Signature verification failed')
        jwt.exceptions.DecodeError: Signature verification failed
```

**Comment by rodrigc** [comment]
Can you report if the problem still occurs with buildbot 0.9.9.post1?  Some additional py3 fixes have gone in

**Comment by pmatos** [comment]
@rodrigc thanks, that should teach me to try the latest version before reporting an issue. It seems to be fixed in `0.9.9.post2`.

**Comment by tardyp** [comment]
especially for python3, a lot of fixes are done in the master branch and
latest releaes :)
